site/content/en/docs/tasks/run_plain_pods.md

```diff
@@ -105,3 +105,47 @@ You can create the Pod using the following command:
 # Create the pod
 kubectl apply -f kueue-pod.yaml
 ```
+
+## Pod Group definition
+
+In order to run a set of pods as a single unit, called Pod Group, add the
+"pod-group-name" label, and the "pod-group-total-count" annotation to all
+members of the group, consistently:
+
+```yaml
+metadata:
+  labels:
+    kueue.x-k8s.io/pod-group-name: "group-name"
+  annotations:
+    kueue.x-k8s.io/pod-group-total-count: "2"
+```
+
+## Feature limitations
+
+Kueue provides only the minimal required functionallity of running pod groups,
+just for the need of environments where the pods are managed by external
+controllers directly, without a Job-level CRD.
+
+As a consequence of this design decision Kueue does not re-implement core
+functionalities that are available at the Job-level API, such as advanced retry
+policies. In particular, Kueue does not re-create failed pods.
+
+Note that, this design choice impacts the scenario of
+[preemption](/docs/concepts/cluster_queue/#preemption).
+When a workload represented by the pod group is preempted all of its pods
+are killed by Kueue (by delete requests). However, later, when the workload is
+re-admitted, Kueue will not re-create the terminated pods. This task is left to
+the user (or the external controller).
+
+**NOTE:** We recommend migration to using Job-level APIs for managing sets of pods.
```

Comment:
It does not matter if custom in-house CRD or already supported, what matters is that there is the job API object that maintains the pods. I guess we could list some supported integrations, but not sure this would be distraction here. I'm ok either way.

Reply:
I often receive questions: what is `Job level`? Does `Job level` mean batch/job?
So, I'd like to add explanations that the `Job-level` means higher level API (CRD) rather than pods, and it directly or indirectly manages pods.

Also, as you say, putting already supported jobs isn't needed.

Reply:
I agree with @tenzen-y. Maybe something like:

`We recommend using the kubernetes Job API or similar CRDs such as JobSet, MPIJob, RayJob, etc.` and link to the documentation for all supported CRDs.

Reply:
ok, sgtm, adjusting